capacitor.config.json

```diff
@@ -20,5 +29,6 @@
       }
     }
   },
-  "cordova": {}
+  "cordova": {},
+  "linuxAndroidStudioPath": "/opt/android-studio-4.1/android-studio/bin/studio.sh"
```

Comment:
@rizwan2000rm  I think `linuxAndroidStudioPath` is only required in the Linux system so you can remove that thing from PR  :)

but you can add this in the `README.md` file while adding  instructions for a running project on Android devices

Reply:
Yeah I did thought it wasn't necessary to push. Also it has to be different for every system.
Will revert this.